.ci/travis-prepare.sh

```diff
@@ -32,6 +32,8 @@ then
   fi
 fi
 
+cd "$CURDIR"
```

Comment:
This change makes no sense.
I spent a lot of effort to make the scripts the same across all modules. Please don't start making arbitrary changes.